ocf_datapipes/training/pvnet_site.py

```diff
@@ -38,6 +38,8 @@
 xr.set_options(keep_attrs=True)
 logger = logging.getLogger("pvnet_site_datapipe")
 
+
+# TODO, these are used any more, but ive left them in here fo reference
```

Comment:
If they are not used I think we should scrape them. There is no reference to those in the code anymore so it will not help reading, and if needed we will be able to find them in blame

Reply:
Thanks, ive removed it